### ISSUE TITLE
Include accessibility support

### ISSUE THREAD

**Issue by ontruster74** [ontruster74/rancid-tomatillos#24]
Auditory - This might relate to someone who is deaf / has a hearing loss. Imagine trying to watch content such as YouTube without captions!

TODO: Our application is purely visual, and does not have any auditory accessibility concerns.

Visual - This revolves around those that are blind and unable to interact with tech visually. Color blindness is also very common in trying to distinguish the difference between colors such as greens and reds, blues and yellows, as well as complete color blindness.

TODO: Our application should implement WAI-ARIA tags and roles where possible in order to render the site more accessible. Verify that there are no aspects of our web app that rely on the user's perception of color.

Cognitive - This can include any learning disability including dyslexia.

TODO: Consider using Comic Sans or another accessible font throughout the app to accommodate dyslexic users.

Mobility - This can relate to dexterity issues such as the difficulty of typing on a phone, the placement of icons and navigation elements, etc.

TODO: Explore mobile or touchscreen-based implementations of the app for those who have trouble with keyboards. Ensure that the site is fully navigable through keystrokes for those who do not or cannot utilize a mouse.
